### PR TITLE
NOISSUE - Add Message Broker Documentation

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
It adds message broker documentation

### Which issue(s) does this PR fix/relate to?
None

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### Notes
N/A

### CODE REVIEW

File: docs/dev-guide.md [hunk 1]
```diff
@@ -240,6 +240,18 @@ This will install `gnatsd` binary that can be simply run by executing:
...
+To do this you can check out the various methods of installing rabbitmq on your local system [here](https://www.rabbitmq.com/download.html).
+
+If you want to change the default message broker to VerneMQ you need to install it on the local system.
+
+To do this you can check out the various methods of installing VerneMQ on your local system [here](https://vernemq.com/downloads/).
+
+If you want to change the default message broker to Kafka you need to install it on the local system.
+
+To do this you can check out the various methods of installing VerneMQ on your local system [here](https://kafka.apache.org/quickstart).
+
```
Comment (manuio) [inline docs/dev-guide.md]: You can simplify the first sentence with something like `If you want to change the default message broker to RabbitMQ, VerneMQ or Kafka you need to install it on the local system.` and do something similar with the links or even use them directly in the names.

**Review by manuio — CHANGES_REQUESTED** [review]

File: docs/dev-guide.md [hunk 2]
```diff
@@ -240,6 +240,18 @@ This will install `gnatsd` binary that can be simply run by executing:
...
+To do this you can check out the various methods of installing rabbitmq on your local system [here](https://www.rabbitmq.com/download.html).
+
+If you want to change the default message broker to VerneMQ you need to install it on the local system.
+
+To do this you can check out the various methods of installing VerneMQ on your local system [here](https://vernemq.com/downloads/).
+
+If you want to change the default message broker to Kafka you need to install it on the local system.
+
+To do this you can check out the various methods of installing VerneMQ on your local system [here](https://kafka.apache.org/quickstart).
```
Comment (drasko) [inline docs/dev-guide.md]: Installing Kafka

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**